src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -219,11 +227,10 @@ export class ViewMappingProfile extends Component {
     };
 
     return (
-      <Pane
-        id={paneId}
-        defaultWidth="fill"
-        fluidContentWidth
+      <FullScreenView
+        paneId={paneId}
         renderHeader={this.renderPaneHeader}
+        className={sharedCss.fullScreenDetails}
```

Comment:
Please try the approach that ui-data-export uses in order to get rid of this kind of style. I think the usage of Layer can avoid such CSS files. Up to you really.

https://github.com/folio-org/ui-data-export/blob/master/src/settings/ProfileDetails/ProfileDetails.js#L60

Reply:
Thanks! Using Layer helps to avoid additional styles.